scripts/start-orderly-web.sh

```diff
@@ -18,9 +18,10 @@ fi
 if which -a orderly-web > /dev/null; then
   echo "orderly-web already installed"
 else
-  pip3 install orderly-web
+  pip3 install orderly-web --user
 fi
 
+export PATH=$PATH:/var/lib/buildkite-agent/.local/bin
```

Comment:
really we want this to be applied to the agents by default, but this is a temporary fix for this build